train.py

```diff
@@ -50,22 +49,21 @@
 arg_parser.add_argument("--spatial_backbone", type=str, default="mean_pixel")
 arg_parser.add_argument("--temporal_backbone", type=str, default="LSTM")
 arg_parser.add_argument("--image_size", type=int, default=32)
-arg_parser.add_argument("--save_model_validation_threshold", type=float, default=0.8)
-arg_parser.add_argument("--pse_sample_size", type=int, default=64)
-arg_parser.add_argument("--validation_split", type=float, default=0.25)
-arg_parser.add_argument("--skip_bands", dest="include_bands", action="store_false")
-arg_parser.set_defaults(include_bands=True)
-arg_parser.add_argument("--skip_cloud", dest="include_cloud", action="store_false")
-arg_parser.set_defaults(include_cloud=True)
-arg_parser.add_argument("--skip_ndvi", dest="include_ndvi", action="store_false")
-arg_parser.set_defaults(include_ndvi=True)
+arg_parser.add_argument("--save_model_threshold", type=float, default=0.8)
+arg_parser.add_argument("--pse_sample_size", type=int, default=32)
+arg_parser.add_argument("--validation_split", type=float, default=0.2)
+arg_parser.add_argument("--split_by", type=str, default="longitude", help="latitude or longitude")
+arg_parser.add_argument("--include_bands", type=bool, default=True)
+arg_parser.add_argument("--include_cloud", type=bool, default=True)
```

Comment:
I just realized, why are we including clouds by default? I thought, based on the DENETHOR paper, declouded ones worked much better.

`When cloudy and raw Sentinel-2 data
is used, convolutions may struggle to identify the relevant observations. In well-prepared, declouded
images, temporal convolutions seem to excel and may hence be better suited for our dataset.`

Reply:
Perhaps it's not aptly named. If you look at where this is used you'll reach this comment:
> param include_cloud: It includes cloud probabilities in the image_stack if TRUE, otherwise it saves the cloud info as separate array

Essentially we are including a band that indicates cloud percentage here

Reply:
I see, got it.